pq-crypto/bike_r3/aes_ctr_prf.c

```diff
@@ -0,0 +1,97 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0"
+ *
+ * Written by Nir Drucker, Shay Gueron and Dusan Kostic,
+ * AWS Cryptographic Algorithms Group.
+ */
+
+#include "aes_ctr_prf.h"
+#include "utilities.h"
+
+ret_t init_aes_ctr_prf_state(OUT aes_ctr_prf_state_t *s,
+                             IN const uint32_t        max_invokations,
+                             IN const seed_t *seed)
+{
+  if(0 == max_invokations) {
+    BIKE_ERROR(E_AES_CTR_PRF_INIT_FAIL);
+  }
+
+  // Set the key schedule (from seed).
+  // Make sure the size matches the AES256 key size.
+  DEFER_CLEANUP(aes256_key_t key, aes256_key_cleanup);
+
+  bike_static_assert(sizeof(*seed) == sizeof(key.raw), seed_size_equals_ky_size);
+  bike_memcpy(key.raw, seed->raw, sizeof(key.raw));
+
+  GUARD(aes256_key_expansion(&s->ks, &key));
```

Comment:
Safety macros need to use the prefix of the return value. In this case it would be `POSIX_GUARD`. Can you run [`./scripts/s2n_safety_explicit_context/codemod.sh`](https://github.com/aws/s2n-tls/blob/main/scripts/s2n_safety_explicit_context/codemod.sh) script to make sure the code is using the proper conventions?

Reply:
Getting this failure, am I doing something wrong here?
```
aweibel@uf06db69e6c8d5e$ ./scripts/s2n_safety_explicit_context/codemod.sh                                                                                                                                                                [1] 
Requirement already satisfied: codemod in ./scripts/s2n_safety_explicit_context/.venv/lib/python3.6/site-packages (from -r scripts/s2n_safety_explicit_context/requirements.txt (line 1))
++ join api cmake codebuild coverage docker-images docs error lib libcrypto-build libcrypto-root scram scripts utils/s2n_safety.h utils/s2n_safety_macros.h
++ local IFS=,
++ echo api,cmake,codebuild,coverage,docker-images,docs,error,lib,libcrypto-build,libcrypto-root,scram,scripts,utils/s2n_safety.h,utils/s2n_safety_macros.h
+ IGNORES=api,cmake,codebuild,coverage,docker-images,docs,error,lib,libcrypto-build,libcrypto-root,scram,scripts,utils/s2n_safety.h,utils/s2n_safety_macros.h
+ map BAIL_PTR PTR_BAIL
+ mod '( +)BAIL_PTR *\(' '\1PTR_BAIL('
+ codemod --exclude-paths api,cmake,codebuild,coverage,docker-images,docs,error,lib,libcrypto-build,libcrypto-root,scram,scripts,utils/s2n_safety.h,utils/s2n_safety_macros.h --extensions c,h,patch '( +)BAIL_PTR *\(' '\1PTR_BAIL('
Searching for first instance...
Traceback (most recent call last):
  File "/home/ANT.AMAZON.COM/aweibel/workspace/github/s2n/scripts/s2n_safety_explicit_context/.venv/bin/codemod", line 11, in <module>
    load_entry_point('codemod==1.0.0', 'console_scripts', 'codemod')()
  File "/home/ANT.AMAZON.COM/aweibel/workspace/github/s2n/scripts/s2n_safety_explicit_context/.venv/lib/python3.6/site-packages/codemod/base.py", line 1025, in main
    run_interactive(**options)
  File "/home/ANT.AMAZON.COM/aweibel/workspace/github/s2n/scripts/s2n_safety_explicit_context/.venv/lib/python3.6/site-packages/codemod/base.py", line 167, in run_interactive
    for patch in suggestions:
  File "/home/ANT.AMAZON.COM/aweibel/workspace/github/s2n/scripts/s2n_safety_explicit_context/.venv/lib/python3.6/site-packages/codemod/base.py", line 442, in generate_patches
    lines = list(open(path))
  File "/usr/lib/python3.6/codecs.py", line 321, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xb0 in position 3: invalid start byte

```

Reply:
Hmm... it looks like some of your files might not be valid UTF-8? I'm not totally sure which one it would be from the error, though.

Reply:
I made the `GUARD()`=>`POSIX_GUARD()` changes manually